README.md

```diff
@@ -1915,20 +1861,24 @@ Other Style Guides
   <a name="blocks--braces"></a><a name="16.1"></a>
   - [16.1](#blocks--braces) Use braces with all multi-line blocks.
 
+    **NEEDS REVIEW**
+
+    > Concern: We should support single-line `if` statements. Currently, the styling of these blocks is ambiguous. Always use braces is the most popular solution.
+
     ```javascript
     // bad
     if (test)
       return false;
 
     // good
-    if (test) return false;
+    if (test) { return false; }
 
     // good
     if (test) {
       return false;
     }
 
-    // bad
```

Comment:
yea i also don't understand why this is bad; this is useful inside of CPs for defining simple `get` and `set` functions.